### DISCUSSION TITLE
Can not login to LTSP client

### DISCUSSION THREAD

**Discussion by isobelle2998** [ltsp/ltsp#256] [Q&A]
I have successfully installed and configured LTSP server and I have netboot a client.
The problem is that the username name on the client is the same as the username on my LTSP server, but when I am enterring the password that correspond to that user. I am told that the login is incorrect.

**Comment by alkisg** [comment]
Add a new user on the server. Run `ltsp initrd`.
Then reboot the client and log in with that new user.

User accounts and /home come from the server; only the rootfs comes from the image.

**Comment by isobelle2998** [comment]
I did as you said. The new user was presented to me in the login page. But I still get the message "Login incorrect"

**Comment by alkisg** [comment]
Press Ctrl+Alt+F2 and try to login there in vt2.
See if you get a longer message explaining more.

For example, you might have forgotten to install ltsp and sshfs in the VM.

**Comment by isobelle2998** [comment]
I am on a virtual machine in virt-manager and Ctrl+Alt+F2 does nothing.

**Comment by alkisg** [comment]
Google for how to change VT in virt-manager/kvm/qemu.

**Comment by isobelle2998** [comment]
And when you say that for example I might have forgotten to install ltsp and sshfs in the VM, which VM are you talking about. Is it the VM that I am using as image? Was I suppose to install the both ltsp and sshfs on it?

**Comment by alkisg** [comment]
Yes, read https://ltsp.org/docs/installation/

**Comment by isobelle2998** [comment]
I did not do it indeed. my bad
Let me do it. And I will be back

**Comment by isobelle2998** [comment]
Everything worked fine. Thank you for the help.

**Comment by alkisg** [comment]
You're welcome. :smile_cat:
